emails/2023-05-04.md

```diff
@@ -0,0 +1,18 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Short and simple this week because I want to keep the message simple. No weekly joke. No three sections. Just a release announcement.
+
+[![Dolt 1.0](../images/dolt-1_0.png)](https://www.dolthub.com/blog/2023-05-05-dolt-1-dot-0/)
+
+Today, we launched [Dolt 1.0](), signaling that the world's first and only version controlled SQL database is ready for your production workloads. If you've been waiting to try Dolt for it to be stable, fast and compatible, now is the time to [try it](https://docs.dolthub.com/introduction/getting-started/database).
```

Comment:
Missing link

Reply:
We don't have it yet. Will add tomorrow.